js/add-pins-map.js

```diff
@@ -0,0 +1,68 @@
+import {addDisabledStatePage, removeDisabledStatePage} from './toggle-state-page.js';
+import {CENTER_TOKYO, PinsOfMap, SIMILAR_AD_COUNT} from './data.js';
+
+const validationForm = document.querySelector('.ad-form');
+
+const buttonReset = validationForm.querySelector('.ad-form__reset');
+
+const addressValue = validationForm.querySelector('#address');
+
+addDisabledStatePage();
+
+const map = L.map('card-canvas')
+  .on('load', () => {
+    removeDisabledStatePage();
+  })
+  .setView(CENTER_TOKYO, 13);
+
+L.tileLayer(
+  'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+  {
+    attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+  },
+).addTo(map);
+
+const mainPinIcon = L.icon(PinsOfMap.MAIN_ICON);
+const generalPinIcon = L.icon(PinsOfMap.GENERAL_ICON);
+
+const addGeneralPinMarker = function (ads) {
+  ads.forEach((item) => {
+    const PinMarker = L.marker(item.location, {
```

Comment:
```suggestion
    const pinMarker = L.marker(item.location, {
```

ну и дальше поправь её, это же просто переменная - они пишутся camelCasом